samples/games/kotlin/src/main/kotlin/godot/samples/games/shmup/EnemyPath.kt

```diff
@@ -12,8 +12,8 @@ class EnemyPath: Path2D() {
         val tween = Tween().apply {
             interpolateProperty(follow, NodePath("unit_offset"), Variant(0), Variant(1), 6.0, Tween.TRANS_LINEAR.toLong(), Tween.EASE_IN_OUT.toLong())
             setRepeat(true)
-            start()
```

Comment:
Seems like someone has gone through some effort to keep this inside an `apply` block.
We could put `addChild(this)` and `start()` inside here as well. No biggie though.

Reply:
I thought of that as well but I think the `addChild` would be out of context here and has nothing to do with `apply` which in my opinion suggests that the block is applying something to the tween.

Reply:
True!

Reply:
Yes call start here was creating a bug, causing the tween to not start. It comes from Godot 3.1 if I remember well.